packages/medusa/src/api/admin/orders/validators.ts

```diff
@@ -52,11 +52,7 @@ export const AdminArchiveOrder = z.object({
 })
 export type AdminArchiveOrderType = z.infer<typeof AdminArchiveOrder>
 
-export type AdminCompleteOrderType = z.infer<typeof CompleteOrder>
-export const CompleteOrder = z.object({
-  order_id: z.string(),
-})
-export const AdminCompleteOrder = WithAdditionalData(CompleteOrder)
+export const AdminCompleteOrder = WithAdditionalData(z.object({}))
```

Comment:
Just FYI, this is a breaking change as now the validation will fail if you pass `order_id` if I am not mistaken

Reply:
True, @olivermrbl how should we label this as a breaking change?

Reply:
we don't use this anywhere yet and doubt any existing users will as well

Reply:
Yeah, should be OK. I'll make a minor note of it in the release notes 👍